docker/vllm_metax.Dockerfile

```
@@ -0,0 +1,164 @@
+FROM registry.access.redhat.com/ubi9/ubi:9.6
+
+# 1. Installing Metax-Driver
+RUN echo "[metax-centos]" > /etc/yum.repos.d/metax-driver-centos.repo && \
+    echo "name=Maca Driver Yum Repository" >> /etc/yum.repos.d/metax-driver-centos.repo && \
+    echo "baseurl=https://repos.metax-tech.com/r/metax-driver-centos-$(uname -m)/" >> /etc/yum.repos.d/metax-driver-centos.repo && \
+    echo "enabled=1" >> /etc/yum.repos.d/metax-driver-centos.repo && \
+    echo "gpgcheck=0" >> /etc/yum.repos.d/metax-driver-centos.repo
```

Might be worth it to make this more readable:

```suggestion
RUN printf "[metax-centos]\n\
    name=Maca Driver Yum Repository\n\
    baseurl=https://repos.metax-tech.com/r/metax-driver-centos-$(uname -m)/\n\
    enabled=1\n\
    gpgcheck=0" >> /etc/yum.repos.d/metax-driver-centos.repo
```

Or to just add `metax-driver-centos.repo` to the repo and `COPY` it in the image